int32 leftTicks
int32 rightTicks
